### PR TITLE
kola/tests/coretest/core: add RHCOS checks for disabled services

### PR DESCRIPTION
**Pull request by miabbott**
In RHCOS, we are explicitly disabling certain services.  These
additional tests will verify that we don't regress.

### CODE REVIEW

File: kola/tests/coretest/core.go [hunk 1]
```diff
@@ -275,6 +296,49 @@ func servicesActive(units []string) error {
 	return nil
 }
 
+func TestServicesDisabledRHCOS() error {
+	inactiveErr := servicesInactive(offServices)
```
Comment (arithx) [inline kola/tests/coretest/core.go]: No need to use separate error variables as you're not returning both, just re-use a single `err` variable.

File: kola/tests/coretest/core.go [hunk 2]
```diff
@@ -275,6 +296,49 @@ func servicesActive(units []string) error {
...
+		}
+	}
+	return nil
+}
+
+func servicesDisabled(units []string) error {
+	for _, unit := range units {
+		c := exec.Command("systemctl", "is-enabled", unit)
+		var out bytes.Buffer
+		c.Stdout = &out
```
Comment (arithx) [inline kola/tests/coretest/core.go]: Any reason to do this rather than `out, err := c.Output()`?

Comment (miabbott) [reply kola/tests/coretest/core.go]: Just learning my way around golang...the example from the docs had this way, but I like the one-line approach much better.

**Comment by miabbott** [comment]
Updated based on review :arrow_up:

**Comment by miabbott** [comment]
Good to merge

Thanks for the feedback @arithx

**Pull request merged**